libraries/Microsoft.Bot.Builder.AI.QnA/Utils/GenerateAnswerUtils.cs

```diff
@@ -2,13 +2,15 @@
 // Licensed under the MIT License.
 
 using System;
```

Comment:
GenerateAnswerUtils.cs and LanguageServiceUtils.cs have many identical methods. Instead of introducing so much code duplication, why not add the methods you need to GenerateAnswerUtils.cs?